### PR TITLE
Add GetCmdArgFloat(Ex) stocks

### PR DESCRIPTION
**Pull request by JoinedSenses**
Adding since there are stocks to handle int values but nothing for float.

### CODE REVIEW

**Comment by JoinedSenses** [comment]
image

**Review by Headline — APPROVED** [review]
Yeah I think this is great and makes sense, thank you!

**Comment by MartLegion** [comment]
Yeah is useful for me as well, sometimes, as JS mentioned, I create commands in plugins to set some netprops. 
As a use case, e.g. in L4D2 you can set a burning percentage from 0.0 to 1.0 to change the Special Infected appearence)
Where 0.5 means 50%, etc.

File: plugins/include/console.inc [hunk 1]
```diff
@@ -443,17 +444,51 @@ stock int GetCmdArgInt(int argnum) {
...
+ * Retrieves a float command argument given its index, from the current
+ * console or server command. Will return 0.0 if the argument can not be
+ * parsed as a number. Use GetCmdArgFloatEx to handle that explicitly.
+ *
+ * @param argnum        Argument number to retrieve.
+ * @return              Value of the command argument.
+ */
+stock float GetCmdArgFloat(int argnum)
+{
+    char str[12];
```
Comment (asherkin) [inline plugins/include/console.inc:469]: Some quick research indicates this buffer is likely too small for reasonable floats, it's hard to find a good reference but 17 might be a better value.

**Review by asherkin — CHANGES_REQUESTED** [review]

File: plugins/include/console.inc (hunk 1 above)
Comment (JoinedSenses) [reply plugins/include/console.inc]: Made it 18 because why not

**Review by asherkin — APPROVED** [review]

**Pull request merged**